### PR TITLE
2 add site footer

### LINKED ISSUE #2: Add: site footer
Is your feature request related to a problem? Please describe.
N/A

Describe how the solution should behave
The footer component should be displayed on all pages of the website, the component will be in the `components` directory in the `src` directory and imported into the `app.js`. All the links in the footer should reflect real pages on the website.

Screenshots of the design/if any
image

Additional context
react-router-dom is already imported to the repo.
All routes should be a constant variable stored in the `src/routes.js` file and exported from there for ease of use.

### PR DESCRIPTION
**Pull request by ZaminMirzad**
- added a notfound page 
- i pulled the navbar branch to get the navbar setups

image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,57 +1,31 @@
...
+import About from './pages/About';
+import Contact from './pages/Contact';
+import Login from './pages/Login';
+import SignUp from './pages/SignUp';
+import NotFound from './pages/NotFound';
+import Footer from './components/Footer';
+
+function App() {
+  return (
+    <div className=" ">
```
Comment (MuhannedNoman) [inline src/App.jsx]:
```suggestion
    <div>

```

File: src/assets/github.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...c2.166,0,4.243-0.332,6.19-0.984C29.585,43.127,29,42.641,29,42.002v-5.804	c0-1.329-0.527-2.53-1.373-3.425C33.473,32.071,36.744,29.405,36.744,23.334z M11.239,32.727c-0.154-0.079-0.237-0.225-0.185-0.328	c0.052-0.103,0.22-0.122,0.374-0.043c0.154,0.079,0.237,0.225,0.185,0.328S11.393,32.806,11.239,32.727z M12.451,33.482	c-0.081,0.088-0.255,0.06-0.389-0.062s-0.177-0.293-0.096-0.381c0.081-0.088,0.255-0.06,0.389,0.062S12.532,33.394,12.451,33.482z M13.205,34.732c-0.102,0.072-0.275,0.005-0.386-0.15s-0.118-0.34-0.016-0.412s0.275-0.005,0.386,0.15	C13.299,34.475,13.307,34.66,13.205,34.732z M14.288,35.673c-0.069,0.112-0.265,0.117-0.437,0.012s-0.256-0.281-0.187-0.393	c0.069-0.112,0.265-0.117,0.437-0.012S14.357,35.561,14.288,35.673z M15.312,36.594c-0.213-0.026-0.371-0.159-0.353-0.297	c0.017-0.138,0.204-0.228,0.416-0.202c0.213,0.026,0.371,0.159,0.353,0.297C15.711,36.529,15.525,36.62,15.312,36.594z M16.963,36.833c-0.227-0.013-0.404-0.143-0.395-0.289c0.009-0.146,0.2-0.255,0.427-0.242c0.227,0.013,0.404,0.143,0.395,0.289	C17.381,36.738,17.19,36.846,16.963,36.833z M18.521,36.677c-0.242,0-0.438-0.126-0.438-0.281s0.196-0.281,0.438-0.281	c0.242,0,0.438,0.126,0.438,0.281S18.762,36.677,18.521,36.677z"/></svg>
```
Comment (MuhannedNoman) [inline src/assets/github.svg]: Move into another folder called `images` inside the `assets` directory.

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import { Link } from 'react-router-dom';
+import logo from '../assets/logov.png';
+import Linkedin, { Facebook, Github } from './Icons';
```
Comment (MuhannedNoman) [inline src/components/Footer.js]:
```suggestion
import { Link } from 'react-router-dom';

import Linkedin, { Facebook, Github } from './Icons';
import logo from '../assets/logov.png';
```

File: src/components/Footer.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            <Linkedin />
+            <Facebook />
+            <Github />
+          </div>
+        </div>
+        {/* menu section */}
+        <div className="flex flex-col w-80 px-5 gap-1 sm:w-80 sm:px-5 ">
+          <h1 className="text-white text-xl mb-4">MENU</h1>
+          <Link
+            to="/"
```
Comment (MuhannedNoman) [inline src/components/Footer.js]: Let's use the `routes.js` file and expose those routes as constant and use them once we need them.

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
Hey @ZaminMirzad nice work here as well. I did not repeat my comments from the previous PR but added some for this PR only. 
Also, if possible remove the Nav from this PR, and let's focus on it in the other PR.

File: src/components/Footer.js (hunk 4 above)
Comment (zaminmirzad) [reply src/components/Footer.js]: routes.js is used now

File: src/assets/github.svg (hunk 2 above)
Comment (zaminmirzad) [reply src/assets/github.svg]: images moved to images folder

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
Hey @ZaminMirzad This looks good, just add the snapshot tests and good to go.

**Review by MuhannedNoman — APPROVED** [review]

**Comment by ZaminMirzad** [comment]
Thanks for the review
I read the comments and will fix as soon as possible

**Pull request merged**